documentation/asciidoc/topics/attributes/community-attributes.adoc

```diff
@@ -6,7 +6,6 @@
 // Spring Boot attributes
 //
 :sb_repo: git@github.com:infinispan/infinispan-spring-boot.git
-:sb_starter: ${version.infinispan.starter}
```

Comment:
set it to 2.1.10.Final ? and change the name to `sb2_starter`? WDYT?

Reply:
I think the attribute to change is "sb_15_starter" because that is unique to DG with support for Spring 1.5. make sense?

Reply:
yes